crypto/bf/bf_local.h

```diff
@@ -10,6 +10,7 @@
 #ifndef OSSL_CRYPTO_BF_LOCAL_H
 # define OSSL_CRYPTO_BF_LOCAL_H
 # include <openssl/opensslconf.h>
+#include <internal/common.h>
 
 /* NOTE - c is not incremented as per n2l */
 # define n2ln(c,l1,l2,n) { \
```

Comment:
This one and the ones below should also go, no? They aren't guarded, so there's a redefinition here.